packages/react/src/Grid/GridCell.tsx

```diff
@@ -34,10 +35,19 @@ export const gridColumnClassNames = (gridColumns?: GridColumns) => {
 
 export const GridCell = forwardRef(
   (
-    { children, gridColumns, className, ...restProps }: PropsWithChildren<GridCellProps>,
+    { children, fullWidth, gridColumns, className, ...restProps }: PropsWithChildren<GridCellProps>,
     ref: ForwardedRef<HTMLDivElement>,
   ) => (
-    <div {...restProps} ref={ref} className={clsx('amsterdam-grid-cell', gridColumnClassNames(gridColumns), className)}>
+    <div
+      {...restProps}
+      ref={ref}
+      className={clsx(
+        'amsterdam-grid-cell',
```

Comment:
I’ve been going back and forth between adding and removing this class and using `amsterdam-page-grid > …` instead. The latter is a bit more automatic, as grid works on direct children anyway, seems a bit brittle, doesn’t affect specificity much. The explicit class is more in line with other components. Also, we need `amsterdam-grid-cell-full-width` anyway and having it without the basic one feels odd.